game/client_types/player.js

```diff
@@ -69,7 +69,7 @@ module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
                 subdiv = document.createElement('div');
                 div.appendChild(subdiv);
                 bars.createBar(subdiv, contribs[i].contribution,
-                    node.game.settings.COINS, color, text);
+                    (1.3 * node.game.settings.COINS), color, text);
```

Comment:
1.3 should be a parameter taken from node.game.settings, e.g. node.game.settings.MULTIPLIER